src/API/WesternStatesWater.WestDaat.Client.Functions/System.cs

```diff
@@ -24,82 +21,84 @@ public System(ISystemManager systemManager, INotificationManager notificationMan
         private readonly ISystemManager _systemManager;
         private readonly ILogger _logger;
 
-        [FunctionName(nameof(GetBeneficialUses))]
-        public async Task<IActionResult> GetBeneficialUses([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "system/beneficialuses")] HttpRequest req)
+        [Function(nameof(GetBeneficialUses))]
+        public async Task<HttpResponseData> GetBeneficialUses([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "system/beneficialuses")] HttpRequestData req)
         {
             _logger.LogInformation("Getting Beneficial Uses");
 
             var results = await _systemManager.GetAvailableBeneficialUseNormalizedNames();
 
-            return new OkObjectResult(results);
+            return await JsonResult(req, results);
```

Comment:
Calling `JsonResult` will use JsonSerializer that converts Enums to Strings and break the front-end.